lib/Component/VEvent.php

```diff
@@ -52,7 +52,10 @@ public function isInTimeRange(\DateTime $start, \DateTime $end) {
             //
             // See:
             // http://tools.ietf.org/html/rfc5545#page-54
-            $effectiveEnd = $this->DTEND->getDateTime();
+            $effectiveEnd = $this->DTEND->getDateTime($end->getTimezone());
+            if (!$this->DTEND->hasTime()) {
+                $effectiveEnd->modify('-1 second');
```

Comment:
Instead of doing `-1 second`, I think you can just alter the condition in the bottom of the file from a `<=` into a `<`, no?